mantidproject/mantid
# Add new NIMROD IDF to replace old NIM_Definition

**Description of work.**

Users had been using the new definition by adding the new
definition here to the bundle. This adds the IDF provided
by scientists and removes the incorrectly named file.



**To test:**

* Test at ISIS
* Start workbench and load a NIMROD file, e.g. NIMROD66223, (with archive search on).
* Show the workspace details and verify the IDF path is the new file.

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
